scripts/ci-setup

```diff
@@ -27,6 +27,12 @@ do
   shift
 done
 
+# Remove env file to get a clean env
+echo "Removing old .env file"
```

Comment:
I would remove this one and just keep the one on line 33.

Reply:
Opps good catch

Reply:
Fixed in a34864c9aed82291c1c5ce3f10786cab5d0c8389